BrowserKit/Sources/TabDataStore/TabDataStore.swift

```diff
@@ -8,7 +8,7 @@ import Common
 // MARK: Protocol
 public protocol TabDataStore {
     func fetchWindowData() async -> WindowData
-    func saveWindowData(window: WindowData) async
+    func saveWindowDataWithBackup(window: WindowData) async
```

Comment:
This method creating a back up or not is an implementation detail that doesn't need to be exposed to the outside world. I would keep the name as it was before.
I meant to leave this comment in the last review but must have forgot to submit it.